core/arch/arch.h

```diff
@@ -512,6 +512,21 @@ mangle_special_registers(dcontext_t *dcontext, instrlist_t *ilist, instr_t *inst
 #endif
 void mangle_insert_clone_code(dcontext_t *dcontext, instrlist_t *ilist,
                               instr_t *instr _IF_X86_64(gencode_mode_t mode));
+/* Returns the number of bytes the stack pointer has to be aligned to. */
+static inline uint
+get_ABI_stack_alignment() {
+#ifdef X86
+ #if defined(X64) || defined(MACOS)
```

Comment:
style: # in 1st column, here and several instances below

Reply:
done